src/test/java/hw10/BoardApiTest.java

```diff
@@ -0,0 +1,85 @@
+package hw10;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.example.entities.BoardEntity;
+import org.testng.annotations.Test;
+
+public class BoardApiTest extends AbstractApiTest {
+
+    @Test(priority = 1)
+    public void addNewBoard() {
+        expectedBoardName = RandomStringUtils.random(50,true,true);
+        board = given()
+            .spec(reqSpec)
+            .when()
+            .queryParam("name", expectedBoardName)
+            .basePath(endpointBoard)
+            .post()
+            .then()
+            .body("name",equalTo(expectedBoardName) )
+            .log().all()
+            .spec(respSpec)
+            .extract().body().as(BoardEntity.class);
+        System.out.println(board.id());
```

Comment:
for console output use logger, for ex log4j. Or delete it, because its debugging artefact